src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -207,23 +191,12 @@ export default function transformForSubmit(formConfig, form) {
 
   dataPostTransform.supportingDocs = dataPostTransform.supportingDocs
     .flat()
-    .concat(supDocs)
-    .filter(el => el); // remove undefineds
-
-  // eslint-disable-next-line dot-notation
-  dataPostTransform.veteran.address['postal_code'] =
-    dataPostTransform.veteran.address.postalCode || '';
-  delete dataPostTransform.veteran.address.postalCode;
-
+    .concat(supDocs);
   dataPostTransform.certifierRole = transformedData.certifierRole;
   dataPostTransform.statementOfTruthSignature =
     transformedData.statementOfTruthSignature;
```

Comment:
Removed unnecessary filter - `supportingDocs` is always empty at this stage, and any undefined values in `supDocs` have already been removed above. 

Additionally, deleted needless keyname change of `postalCode`.